lib/sockets/pbpal_resolv_and_connect_sockets.c

```diff
@@ -157,9 +164,8 @@ static void get_dns_ip(struct sockaddr* addr)
 static void get_dns_ip(struct sockaddr* addr)
 {
     addr->sa_family = AF_INET;
-    inet_pton(AF_INET,
-              PUBNUB_DEFAULT_DNS_SERVER,
-              &(((struct sockaddr_in*)addr)->sin_addr.s_addr));
+    struct pubnub_ipv4_address* p = &(((struct sockaddr_in*)addr)->sin_addr.s_addr));
```

Comment:
It seems code is using the same function for both IPV4 and IPv6.  Please confirm if this works for IPv6?

Reply:
Yes, it will be fine for IPv6, as long as `8.8.8.8` was always working for IPv6 before this patch! If there were preexisting bugs with that setup, we can address them. I don't think I enabled IPv6 in my testing, but I have full IPv6 support on my network. I can make some additional passes to ensure it's all good.

Reply:
I just learned that IPv6 is already enabled by default in the makefiles. I am satisfied that it works as intended. Would you agree?